.github/workflows/unit_tests.yml

```diff
@@ -22,7 +22,7 @@ env:
 jobs:
   unit-test:
     name: Unit test
-    runs-on: ubuntu-20.04
+    runs-on: [self-hosted, Linux, X64, utest]
```

Comment:
Do we need to move all of the unit tests on self-hosted?

Reply:
I think so, all jobs under `unit-test` include the step `Install PMU tools`

Reply:
Yeah, but only the profile unit test actually needs it. You've tried that in separate PR. I'm thinking about offloading to self-hosted only the parts that cannot be ran on github runners. Just like firecracker unit tests in the same file.

Reply:
Okay, no problem. I will change the yaml structure